src/cursor.rs

```diff
@@ -469,9 +469,12 @@ impl Cursor {
     /// # Return value
     ///
     /// Returns a vector containing the BSON documents that were read.
-    pub fn next_batch(&mut self) -> Result<Vec<bson::Document>> {
-        let n = self.batch_size;
-        self.next_n(n)
```

Comment:
let's use the deprecation feature here, and default to 20 as before? we can suggest clients use `cursor.next_n(cursor.batch_size)` if they need to replicate the exact functionality.